web-registry/src/graphql/MoreProjects.graphql

```diff
@@ -5,7 +5,7 @@ fragment moreProjectFields on Project {
   creditClassByCreditClassId {
```

Comment:
I think we can just drop this entire file too as well as `useOtherProjects` which uses it but is unused

Reply:
it looks like we still need the `MoreProjects` query in `CreditClassDetailsWithContent.tsx`

Reply:
oh right!